### ISSUE TITLE
Displaying of special characters in tables

### ISSUE THREAD

**Issue by Ludee** [OpenEnergyPlatform/oeplatform#1014]
Looking at [this table](https://openenergy-platform.org/dataedit/view/reference/rli_lodgeoss_energy_database_review) it looks like the OEP cannot display certain special characters.

It includes 
- ²
- °
- apostrophes (è, É ....)

**Comment by wingechr** [comment]
I don't think that this is true. if you look at the data directly as json (https://openenergy-platform.org/api/v0/schema/reference/tables/rli_lodgeoss_energy_database_review/rows/), it shows that the invalid characters are in the data itself, i.e. the uploader messed up with character encoding.

I also tested it locally and inserted unicode, and is displays properly:

![grafik](https://user-images.githubusercontent.com/4265368/177268566-6a5defdb-e081-4573-8e08-10e95cff98b8.png)

**Comment by Ludee** [comment]
Thanks @wingechr for testing and solving!
Since I was the uploader I need to check at which point the error occurred.
I will keep this issue open since this information needs to be documented somewhere.
